src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -108,7 +108,7 @@ function ProctoredExamSettings({ courseId, intl }) {
 
   function handleSubmit(event) {
     event.preventDefault();
-    if (proctoringProvider === 'proctortrack' && !EmailValidator.validate(proctortrackEscalationEmail)) {
+    if (proctoringProvider === 'proctortrack' && !EmailValidator.validate(proctortrackEscalationEmail) && enableProctoredExams) {
```

Comment:
We may want this to only accept none (in the case of disabled) or a valid value. I think saving an invalid value will cause errors if someone tries to re enable proctored exams at a later point.

Reply:
Good catch, I'll update this